acceptance/bundle/exec/no-auth/script

```diff
@@ -0,0 +1,7 @@
+unset DATABRICKS_HOST
+unset DATABRICKS_TOKEN
+
+# Confirm that bundle exec works for commands that do not require authentication,
+# even if authentication is not provided.
+trace $CLI bundle exec -- echo hello
+trace $CLI bundle exec -- python3 -c 'import os; print(os.getcwd())'
```

Comment:
Let's have one more line here for a command that does require authentication? Like current-user.

Reply:
Good point, added.